add Republic Of Gamers icon

VikashPR opened the pull request:
<!--
Before opening your pull request, have a quick look at our contribution guidelines:
https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md

Consider adding a preview image of your submission using:
https://petershaggynoble.github.io/SI-Sandbox/preview/
-->

**Issue:**
**Alexa rank:**
  <!-- The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/
       Please see our contributing guidelines for more details on how we
       assess a brand's popularity. -->
816
### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
<!--
Anything relevant, for example:
  - Why did you pick the hex value?
  - Did you manually vectorize the logo?
  - Have you used multiple sources?
  - etc.
-->
The hex values are taken from the official website.

sachinraja commented on _data/simple-icons.json:
```diff
@@ -11622,6 +11622,17 @@
             "source": "https://www.zyte.com/"
+        },
+        {
+            "title": "Asus ROG",
```
This data is in alphabetical order

vikash2806 replied:
I have arranged the entire data in alphabetical order, next to ASUS data.

sachinraja commented on icons/asusrog.svg:
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><title>ASUS ROG</title><path fill-rule="evenodd" clip-rule="evenodd" d="M14.4162 1.76646C14.3788 1.77737 14.369 1.79409 14.3319 1.8257C14.302 1.85113 14.2767 1.86919 14.2477 1.89404C14.1812 1.95095 14.1422 1.98614 14.0838 2.03499C13.7937 2.27761 13.4001 2.6556 13.1157 2.92032C12.6746 3.33097 12.2771 3.72644 11.8538 ...
```
extra path attributes like `fille-rule` and `clip-rule` are not allowed

vikash2806 replied:
I've removed those attributes.

sachinraja replied:
I shouldn't have called them extra, these attributes often change the icon in some way and it's not as simple as just removing them. Might have to call another maintainer in because I have had trouble with removing these attributes before.

sachinraja reviewed (changes requested):
requested changes above

sachinraja reviewed (changes requested):
Linter is failing please read errors: https://github.com/simple-icons/simple-icons/pull/7124/checks#step:6:26 and read [this section in the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon).

vikash2806 commented:
> Linter is failing please read errors: https://github.com/simple-icons/simple-icons/pull/7124/checks#step:6:26 and read [this section in the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon).

There is a precision error while running lint, I will try re-uploading new SVG. 
is there something else I can do to solve that error ?

sachinraja commented:
Icons must be [optimized](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon). Again, please read the guidelines.

vikash2806 commented:
> [Icons must be optimized](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon). Again, please read the guidelines.

Hey can you please review my new changes?
I've added new icon and it passed all 3 checks.

sachinraja reviewed (changes requested):
Looks like you sized the logo to be 24 x 24 px. We want the logo to look exactly the same as the original logos, so we only require that one of the dimensions be 24 px. In this case, since the width is bigger than the height, you would only set the width to 24 px and have the height scale accordingly.

VikashPR commented:
@sachinraja Hope everything is fine this time.

sachinraja commented:
Can you show me the actual icon that you used? I'm seeing quite a big difference with the red icon from the navbar at the top of the page:
![image](https://user-images.githubusercontent.com/58836760/153774754-dd84f179-4910-4c99-890d-bb6c906cc518.png)

VikashPR commented:
> Can you show me the actual icon that you used? I'm seeing quite a big difference with the red icon from the navbar at the top of the page:
> ![image](https://user-images.githubusercontent.com/58836760/153774754-dd84f179-4910-4c99-890d-bb6c906cc518.png)

### This is icon of reference from official page

![img](https://user-images.githubusercontent.com/69889418/153791096-8139fb39-3896-48dd-9142-fe5ec959f8c6.png)

### These are the resources of my Icon:
https://www.svgrepo.com/svg/303479/asus-rog-1-logo
https://worldvectorlogo.com/logo/asus-rog-1
https://iconape.com/rog-republic-of-gamers-logo-logo-icon-svg-png.html

VikashPR commented:
@sachinraja Let me know if you still have any queries.

LitoMore commented on _data/simple-icons.json:
```diff
@@ -8458,6 +8458,17 @@
+                ]
+            },
+            "hex": "FF0029",
+            "source": "https://rog.asus.com/in/"
```
```suggestion
            "source": "https://rog.asus.com"
```

LitoMore commented:
You should never do modifications to their trademark.

VikashPR commented:
> You should never do modifications to their trademark.

what do you mean ?

sachinraja commented:
> These are the resources of my Icon:
svgrepo.com/svg/303479/asus-rog-1-logo
worldvectorlogo.com/logo/asus-rog-1
iconape.com/rog-republic-of-gamers-logo-logo-icon-svg-png.html

Unfortunately, none of these resources are official so we cannot consider them. Could you remake the icon based on the one in their navbar, which is an official svg source:
![image](https://user-images.githubusercontent.com/58836760/154176355-a9b5ea9b-a72e-4b76-a1d5-8ccb7879e773.png)
(svg:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 37" svg-inline="" src="@images/logo/logo-rog.svg" alt="ROG - Republic of Gamers" role="presentation" focusable="false" class="icon"><path d="M22.2 29.1c.8.5 1.6 1 2.4 1.4 8.6 3.8 21.3 6.1 23.7 5.3C55 33.6 62.5 19.5 64.8 13c0 0-7 2.8-14.1 6.1-5.9 2.8-12.4 6-12.4 6l16.5-5s-4.4 10.4-10.2 11.6c-5.8 1.2-16-2.9-16-2.9.8-.7 11.6-8.6 36.8-18.9 1.1-.8 1.8-2.1 2-3.4-5.8.6-11.4 2.3-16.7 4.8-10.6 5-28.5 17.8-28.5 17.8zM3.8 24c1.8 2.3 9.2 4.2 10.2 4.4-4.6-3.2-14-12-14-12 1 2.7 2.3 5.2 3.8 7.6z"></path><path d="M25.6 22.9C30.5 19.3 47.9 4.6 64.7.6c0 0-8.8-1.4-21.3 2.5-4.5 1.4-11 7.7-23.1 19.4-1.6.9-7.5-2.5-10.9-4.1 0 0 5.6 8.9 7.6 11.5 3 4.1 8.5 6.5 8.5 6.5-.5-.5-4.3-4.5-5.4-6.8-.7-1.1.1-2.7 5.5-6.7zM127.9 6.8h-3v8.9h-5.6V6.8h-3v10.4h11.6zM152.9 15.6h-7.5V6.8h-3.1v10.4h9.2zM154.2 6.8h3.1v10.4h-3.1zM191.7 8.3h6.9l1.4-1.5h-11.4v10.4h3.1v-5.1h6.7l1.2-1.6h-7.9zM106.6 13.8h8.3V7.6c0-.4-.3-.8-.7-.8h-10.7v10.3h3.1v-3.3zm0-5.5h5c.1 0 .3 0 .3.5v3.5h-5.3v-4zM140.7 10.8h-.9v-3c0-.6-.3-1-.8-1h-9.7v10.4h11.3l.1-6.4zm-3.1 4.8h-5.2v-3.3h5.2v3.3zm0-4.8h-5.2V8.3h4.9c.3 0 .3.4.3.5v2zM100.5 8.3l1.6-1.5H90.5v10.3h11.6l-1.6-1.5h-6.9v-2.8h7.8v-1.7h-7.8V8.3zM162 8.3h7l1.5-1.5h-11.6v10.3H169l1.5-1.5H162zM154 24.5l1.3-1.5h-11.8v5.6h8.5v3.1h-8.9l1.9 1.6h10.2V27h-8.6v-2.5zM100.6 23h-11v10.3h3.1v-4.4h5.5v4.4h3.1v-9.4c.1-.4-.2-.8-.7-.9zm-2.4 4.4h-5.4v-2.9h5.1c.3 0 .3.3.3.5v2.4zM127.2 24.5l1.6-1.5h-11.7v10.3h11.7l-1.6-1.5h-7V29h7.9v-1.7h-7.9v-2.8zM86.4 24.6L88 23H75.8v10.3h11.8v-5.6h-3.1v4.1H79v-7.2zM103 23v10.4h3.2v-6l3 2.1 3.1-2.1v6h3.1V23l-6.2 4.1zM141.8 28.9v-5c0-.5-.3-1-.8-1h-11.2v10.4h3.1v-8.8h5.4c.3 0 .3.3.3.5v2h-5.1l5.6 6.3h3.7l-3.9-4.4h2.9zM88.7 14.1V7.8c0-.5 0-1-.6-1H75.8V14l3.2 3.5V8.3h6c.4 0 .5.1.5.5v3.3h-5.3l8.5 8.9v-3.5L85.5 14h3.2zM186.1 6.8h-10.5v7.5l2.5 2.9h8.7V7.6c0-.4-.3-.8-.7-.8zm-2.4 8.8h-3.3l-1.7-2V8.3h5v7.3z"></path></svg>
```
)

VikashPR commented:
> > These are the resources of my Icon:
> > svgrepo.com/svg/303479/asus-rog-1-logo
> > worldvectorlogo.com/logo/asus-rog-1
> > iconape.com/rog-republic-of-gamers-logo-logo-icon-svg-png.html
> 
> Unfortunately, none of these resources are official so we cannot consider them. Could you remake the icon based on the one in their navbar, which is an official svg source: ![image](https://user-images.githubusercontent.com/58836760/154176355-a9b5ea9b-a72e-4b76-a1d5-8ccb7879e773.png) (svg:
> 
> ```
> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 37" svg-inline="" src="@images/logo/logo-rog.svg" alt="ROG - Republic of Gamers" role="presentation" focusable="false" class="icon"><path d="M22.2 29.1c.8.5 1.6 1 2.4 1.4 8.6 3.8 21.3 6.1 23.7 5.3C55 33.6 62.5 19.5 64.8 13c0 0-7 2.8-14.1 6.1-5.9 2.8-12.4 6-12.4 6l16.5-5s-4.4 10.4-10.2 11.6c-5.8 1.2-16-2.9-16-2.9.8-.7 11.6-8.6 36.8-18.9 1.1-.8 1.8-2.1 2-3.4-5.8.6-11.4 2.3-16.7 4.8-10.6 5-28.5 17.8-28.5 17.8zM3.8 24c1.8 2.3 9.2 4.2 10.2 4.4-4.6-3.2-14-12-14-12 1 2.7 2.3 5.2 3.8 7.6z"></path><path d="M25.6 22.9C30.5 19.3 47.9 4.6 64.7.6c0 0-8.8-1.4-21.3 2.5-4.5 1.4-11 7.7-23.1 19.4-1.6.9-7.5-2.5-10.9-4.1 0 0 5.6 8.9 7.6 11.5 3 4.1 8.5 6.5 8.5 6.5-.5-.5-4.3-4.5-5.4-6.8-.7-1.1.1-2.7 5.5-6.7zM127.9 6.8h-3v8.9h-5.6V6.8h-3v10.4h11.6zM152.9 15.6h-7.5V6.8h-3.1v10.4h9.2zM154.2 6.8h3.1v10.4h-3.1zM191.7 8.3h6.9l1.4-1.5h-11.4v10.4h3.1v-5.1h6.7l1.2-1.6h-7.9zM106.6 13.8h8.3V7.6c0-.4-.3-.8-.7-.8h-10.7v10.3h3.1v-3.3zm0-5.5h5c.1 0 .3 0 .3.5v3.5h-5.3v-4zM140.7 10.8h-.9v-3c0-.6-.3-1-.8-1h-9.7v10.4h11.3l.1-6.4zm-3.1 4.8h-5.2v-3.3h5.2v3.3zm0-4.8h-5.2V8.3h4.9c.3 0 .3.4.3.5v2zM100.5 8.3l1.6-1.5H90.5v10.3h11.6l-1.6-1.5h-6.9v-2.8h7.8v-1.7h-7.8V8.3zM162 8.3h7l1.5-1.5h-11.6v10.3H169l1.5-1.5H162zM154 24.5l1.3-1.5h-11.8v5.6h8.5v3.1h-8.9l1.9 1.6h10.2V27h-8.6v-2.5zM100.6 23h-11v10.3h3.1v-4.4h5.5v4.4h3.1v-9.4c.1-.4-.2-.8-.7-.9zm-2.4 4.4h-5.4v-2.9h5.1c.3 0 .3.3.3.5v2.4zM127.2 24.5l1.6-1.5h-11.7v10.3h11.7l-1.6-1.5h-7V29h7.9v-1.7h-7.9v-2.8zM86.4 24.6L88 23H75.8v10.3h11.8v-5.6h-3.1v4.1H79v-7.2zM103 23v10.4h3.2v-6l3 2.1 3.1-2.1v6h3.1V23l-6.2 4.1zM141.8 28.9v-5c0-.5-.3-1-.8-1h-11.2v10.4h3.1v-8.8h5.4c.3 0 .3.3.3.5v2h-5.1l5.6 6.3h3.7l-3.9-4.4h2.9zM88.7 14.1V7.8c0-.5 0-1-.6-1H75.8V14l3.2 3.5V8.3h6c.4 0 .5.1.5.5v3.3h-5.3l8.5 8.9v-3.5L85.5 14h3.2zM186.1 6.8h-10.5v7.5l2.5 2.9h8.7V7.6c0-.4-.3-.8-.7-.8zm-2.4 8.8h-3.3l-1.7-2V8.3h5v7.3z"></path></svg>
> ```
> 
> )

@sachinraja the screen shot of image that I've is from official website.
I will add the image once again.
https://rog.asus.com/in/

![Screenshot 2022-02-16 at 6 47 57 AM](https://user-images.githubusercontent.com/69889418/154177828-a31b5d54-0580-43bf-b49c-c65224beeb8a.png)

scroll to the footer of the website you can see ROG logo there.

Even every ROG laptops have the same logo.

https://i0.wp.com/www.smartprix.com/bytes/wp-content/uploads/2020/11/Scar-15-review-1.jpg?resize=696%2C522&ssl=1

sachinraja commented:
I saw the image, but SVG sources are preferred to image sources, so it would be nice if we could base this icon on an SVG.

VikashPR commented:
> I saw the image,

![Screenshot 2022-02-16 at 7 08 32 AM](https://user-images.githubusercontent.com/69889418/154179845-d823a810-4dba-44cf-acd8-1a253a47c645.png)

To be clear there are two different icons. with two path attributes.

sachinraja commented:
You can cut out the `Republic of Gamers` part so only the icon is showing. Also raw SVGs often have multiple paths, but this can easily be fixed by combining them in an editor.

VikashPR commented:
> > I saw the image,
> 
> ![Screenshot 2022-02-16 at 7 08 32 AM](https://user-images.githubusercontent.com/69889418/154179845-d823a810-4dba-44cf-acd8-1a253a47c645.png)
> 
> To be clear there are two different icons. with two path attributes.

@sachinraja 


> You can cut out the `Republic of Gamers` part so only the icon is left. Also raw SVGs often have multiple paths, but this can easily be fixed by combining them in an editor.

alright I will do that I will remove the republic of gamers part from the logo and I will isolate the logo alone.

VikashPR commented:
@sachinraja I've updated the new icon with the help of ROG website logo SVG
![Screenshot 2022-02-16 at 9 45 53 PM](https://user-images.githubusercontent.com/69889418/154308321-9e168e59-6d22-44ef-9349-152c1b1d8df9.png)
https://rog.asus.com
![Screenshot 2022-02-16 at 9 46 26 PM](https://user-images.githubusercontent.com/69889418/154308416-373cbe1f-fc3c-4162-a1f8-dbc0a191453c.png)

As you mentioned there are quite some difference between the old logo and the new logo.
Hopefully everything is alright now. Let me know if there is anything else.

sachinraja commented:
Looking much better! Unfortunately, still seeing minor differences though:
![image](https://user-images.githubusercontent.com/58836760/154398874-d8844af4-1e17-463a-ae43-a33b5d0b245f.png)
(red is original, black is the one in this PR)
You may want to adjust precision when optimizing (we allow a max of 5):
https://github.com/simple-icons/simple-icons/blob/39954116d1eeaeca01143612f369c6a51a3acb56/svgo.config.js#L23

I know these requests for changes may seem pedantic but we want to ensure that we are adding the highest possible quality icons to the library. Thanks for sticking with it and addressing the issues.

VikashPR commented:
> Looking much better! Unfortunately, still seeing minor differences though: ![image](https://user-images.githubusercontent.com/58836760/154398874-d8844af4-1e17-463a-ae43-a33b5d0b245f.png) (red is original, black is the one in this PR) You may want to adjust precision when optimizing (we allow a max of 5):
> 
> https://github.com/simple-icons/simple-icons/blob/39954116d1eeaeca01143612f369c6a51a3acb56/svgo.config.js#L23
> 
> I know these requests for changes may seem pedantic but we want to ensure that we are adding the highest possible quality icons to the library. Thanks for sticking with it and addressing the issues.

alright.

sachinraja commented:
Still not lining up :thinking:
![image](https://user-images.githubusercontent.com/58836760/154420619-1a170050-2a1f-476d-a7a0-85d577831c4c.png)
Might try to optimize it myself

VikashPR commented:
I have no idea why is that happening, may be you should try reducing the red version size to 24,13.

![ROG-Unoptimized](https://user-images.githubusercontent.com/69889418/154426705-ebbe77d6-e924-4614-ac3b-0cd1fc5094f5.svg)

sachinraja commented:
Seeing the same differences with the unoptimized version, you might want to try recreating it? If not, I can try to make it.

> may be you should try reducing the red version size to 24,13

It would be wrong then, we cannot alter the proportions of the source icon.

VikashPR commented:
> Seeing the same differences with the unoptimized version, you might want to try recreating it? If not, I can try to make it.
> 
> > may be you should try reducing the red version size to 24,13
> 
> It would be wrong then, we cannot alter the proportions of the source icon.

Sachin the problem is not recreating the logo, it is about the alignment of logo in view box. When you check my actions testing most of my build failed the lint alignment in like [11.99,12]  instead of [12,12] in decimals but my editor shows it is aligned in [12,12]. The problem comes in alignment, I ran more than 15 tests to get it right.

No problem I will try to recreate the logo once again. can you please share the red logo of yours? I will use that as base file.

sachinraja commented:
> When you check my actions testing most of my build failed the lint alignment in like [11.99,12] instead of [12,12] in decimals but my editor shows it is aligned in [12,12].

Ah ok, thanks for sharing, I never looked at the linter failures. This is usually an issue with precision instead of alignment. When you get that error, you should first try increasing the number of decimals (precision).

VikashPR commented:
> > When you check my actions testing most of my build failed the lint alignment in like [11.99,12] instead of [12,12] in decimals but my editor shows it is aligned in [12,12].
> 
> Ah ok, thanks for sharing, I never looked at the linter failures. This is usually an issue with precision instead of alignment. When you get that error, you should first try increasing the number of decimal places (precision).

I will give it a try. Can you please share the logo which you are using for reference, the red one.

sachinraja commented:
Ah yes, sorry, here it is:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->

<svg
   width="24"
   height="24"
   viewBox="0 0 24 24.000001"
   version="1.1"
   id="svg827"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs824" />
  <g
     id="layer1">
    <path
       id="path821"
       d="m 21.390625,5.5833053 c -1.346761,0.011501 -3.433791,0.1954052 -5.9375,0.9765625 -1.602373,0.4985163 -3.916004,2.7420306 -8.22461,6.9082032 -0.569733,0.320474 -2.670176,-0.891204 -3.880859,-1.460938 0,0 1.992912,3.169888 2.705078,4.095704 1.068249,1.459941 3.027344,2.314453 3.027344,2.314453 -0.178042,-0.178042 -1.530184,-1.602884 -1.921875,-2.421875 -0.249258,-0.391692 0.03418,-0.962387 1.957031,-2.386719 1.744808,-1.281899 7.941633,-6.5151208 13.923828,-7.9394532 0,0 -0.600956,-0.094883 -1.648437,-0.085937 z M 24,7.7708053 C 21.934718,7.9844551 19.939974,8.589582 18.052734,9.4797897 14.278253,11.260205 7.904297,15.81768 7.904297,15.81768 c 0.284866,0.178042 0.570602,0.355614 0.855468,0.498047 3.062315,1.353116 7.584854,2.171585 8.439454,1.886719 2.385756,-0.783383 5.056008,-5.802648 5.875,-8.117187 0,0 -2.493295,0.9968 -5.021485,2.171874 -2.10089,0.997034 -4.414062,2.136719 -4.414062,2.136719 l 5.875,-1.78125 c 0,0 -1.567531,3.70356 -3.632813,4.13086 -2.065282,0.4273 -5.697265,-1.033203 -5.697265,-1.033203 C 10.46846,15.461001 14.313816,12.647446 23.287109,8.9797897 23.678801,8.6949232 23.928783,8.2337133 24,7.7708053 Z M 0,11.294243 c 0.356083,0.961424 0.819391,1.852432 1.353515,2.707031 0.64095,0.818991 3.274777,1.495189 3.63086,1.566406 C 3.346393,14.428214 0,11.294243 0,11.294243 Z" />
  </g>
</svg>
```

VikashPR commented:
> Ah yes, sorry, here it is:
> 
> ```
> <?xml version="1.0" encoding="UTF-8" standalone="no"?>
> <!-- Created with Inkscape (http://www.inkscape.org/) -->
> 
> <svg
>    width="24"
>    height="24"
>    viewBox="0 0 24 24.000001"
>    version="1.1"
>    id="svg827"
>    xmlns="http://www.w3.org/2000/svg"
>    xmlns:svg="http://www.w3.org/2000/svg">
>   <defs
>      id="defs824" />
>   <g
>      id="layer1">
>     <path
>        id="path821"
>        d="m 21.390625,5.5833053 c -1.346761,0.011501 -3.433791,0.1954052 -5.9375,0.9765625 -1.602373,0.4985163 -3.916004,2.7420306 -8.22461,6.9082032 -0.569733,0.320474 -2.670176,-0.891204 -3.880859,-1.460938 0,0 1.992912,3.169888 2.705078,4.095704 1.068249,1.459941 3.027344,2.314453 3.027344,2.314453 -0.178042,-0.178042 -1.530184,-1.602884 -1.921875,-2.421875 -0.249258,-0.391692 0.03418,-0.962387 1.957031,-2.386719 1.744808,-1.281899 7.941633,-6.5151208 13.923828,-7.9394532 0,0 -0.600956,-0.094883 -1.648437,-0.085937 z M 24,7.7708053 C 21.934718,7.9844551 19.939974,8.589582 18.052734,9.4797897 14.278253,11.260205 7.904297,15.81768 7.904297,15.81768 c 0.284866,0.178042 0.570602,0.355614 0.855468,0.498047 3.062315,1.353116 7.584854,2.171585 8.439454,1.886719 2.385756,-0.783383 5.056008,-5.802648 5.875,-8.117187 0,0 -2.493295,0.9968 -5.021485,2.171874 -2.10089,0.997034 -4.414062,2.136719 -4.414062,2.136719 l 5.875,-1.78125 c 0,0 -1.567531,3.70356 -3.632813,4.13086 -2.065282,0.4273 -5.697265,-1.033203 -5.697265,-1.033203 C 10.46846,15.461001 14.313816,12.647446 23.287109,8.9797897 23.678801,8.6949232 23.928783,8.2337133 24,7.7708053 Z M 0,11.294243 c 0.356083,0.961424 0.819391,1.852432 1.353515,2.707031 0.64095,0.818991 3.274777,1.495189 3.63086,1.566406 C 3.346393,14.428214 0,11.294243 0,11.294243 Z" />
>   </g>
> </svg>
> ```

Thanks man, This time I increased the precision to 5 and updated the icon.

sachinraja reviewed (approved):
Awesome, this is a perfect match! Thanks for bearing with me on this.

VikashPR commented:
> Awesome, this is a perfect match! Thanks for bearing with me on this.

Thankyou Sachi, when is the next release ?

sachinraja commented:
We have automatic releases once a week on Saturday or Sunday (this week's release was just done) so this will be out in about a week.
